tests/test_modeling_common.py

```diff
@@ -29,6 +31,16 @@
 
 
 class ModelUtilsTest(unittest.TestCase):
+    def tearDown(self):
+        # clean up the VRAM after each test
+        super().tearDown()
+        gc.collect()
+        torch.cuda.empty_cache()
+
+        import diffusers
+
+        diffusers.utils.import_utils._safetensors_available = True
+
```

Comment:
```suggestion
```

Tied to https://github.com/huggingface/diffusers/pull/2515/files#r1131591497

Reply:
Still want to make sure `diffusers.utils.import_utils._safetensors_available = True` is done in case the test fails